source/SpinalHDL/Getting Started/Install and setup.rst

```diff
@@ -3,44 +3,91 @@
+  get features such as:
+
+    * Code suggestions / completion
+    * Automatic build with syntax errors right in the code
+    * Generate code with a single click
+    * Run simulation / tests with a single click (if a supported simulator is
+      set up)
+
+* A supported simulator like `Verilator <https://www.veripool.org/verilator/>`_
+  to test the design right from SpinalHDL.
+* `Gtkwave <https://gtkwave.sourceforge.net/>`_ to view the waves generated by
+  Verilator during simulation.
+* `Git <https://git-scm.com/>`_ for version control system
+* A C++ toolchain, needed for simulating with Verilator
+* A linux shell, needed for simulating with Verilator
 
 Linux Installation
-----------------------
+------------------
 
-For instance, to install them on the Debian distribution:
+At time of writing the recommended way of installing Scala and SBT is via `Coursier <https://get-coursier.io/docs/cli-installation>`_.
+Coursier is able to in addition to the scala tools install a Java JDK to use, in the
+example below we install Java from the package manager. We recommend to install JDK 17 (LTS)
+because of compatibility with the used Scala version.
+
+For Debian or Ubuntu we run:
 
 .. code-block:: sh
 
-   sudo apt-get install openjdk-8-jdk
-   sudo apt-get install scala
-   echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
-   echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
-   curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | sudo apt-key add
    sudo apt-get update
-   sudo apt-get install sbt
+   sudo apt-get install openjdk-17-jdk-headless curl git
+   curl -fL "https://github.com/coursier/launchers/raw/master/cs-x86_64-pc-linux.gz" | gzip -d > cs
+   chmod +x cs
+   # should find the just installed jdk, agree to cs' questions for adding to your PATH
+   ./cs setup
+   source ~/.profile
+
+If you want to install the tools for simulation and/or formal proofs, we recommend `oss-cad-suite <https://github.com/YosysHQ/oss-cad-suite-build>`_.
+It contains a waveform viewer (gtkWave), verilog simulators (verilator and iverilog), VHDL simulator (GHDL) and other tools.
+In case you want to build the tools yourself have a look at the legacy simulation tool :ref:`installation instructions <sim backend install>`.
+
+We first install the needed C++ toolchain and download oss-cad-suite. To use it we must load the oss-cad-suite
+environment for each shell we want to use it in. Note that oss-cad-suite contains a Python 3 interpreter that
+may interfere with the system Python installation if loaded permanently.
+
+Go to the oss-cad-suite `release page <https://github.com/YosysHQ/oss-cad-suite-build/releases/latest>`_ to get the
+download link for the latest version. You can download/extract oss-cad-suite to a folder of
+your choice.
```

Comment:
How about add a recommended version of oss-cad-suite which has been tested. The lastest tool normally means user should test themselves with time and efforts for what they will never use.

Reply:
will do that